.github/workflows/close-pull-request.yaml

```diff
@@ -0,0 +1,23 @@
+name: PRs to Main
+on: 
+  pull_request:
+    branches: [main]
+    types: [closed]
```

Comment:
I want to make sure that we have a workflow:

1. On PR close (not approved)  - we revert the change to the dev branch.
2. Merge of PRs to main may be automatic but initially it can be manual. We are interested in feedback on the PR from the target test system. If test fails, we should have "changes requested".

Reply:
That is already addressed in `close-pull-request.yaml` this will take care of that when no merged happened and PR closed. For 2, I think lets have it manual for now.